go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.6 // indirect
 	github.com/go-openapi/jsonreference v0.20.2 // indirect
 	github.com/go-openapi/swag v0.22.3 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
```

Comment:
Do we need this dependency ? I thought we are using the ArgoCD Glob.

Reply:
Yes , we are using ArgoCD Glob package. Tt's an indirect dependence, for pkg "github.com/argoproj/argo-cd/v2/util/glob".